SPECS/qemu-kvm/qemu-kvm.spec

```diff
@@ -138,7 +140,50 @@ ln -sv qemu-system-`uname -m` %{buildroot}%{_bindir}/qemu
 chmod 755 %{buildroot}%{_bindir}/qemu
 
 %check
-# Deliberately empty
+testsPassed=true
+make check-unit
+retVal=$?
+if [ $retVal -ne 0 ]; then
```

Comment:
Since we're not using retVal anywhere except these checks we might as well use the $? directly.
 `if [ $? -ne 0 ]; then testsPassed=false; fi`

Reply:
Done